src/reducers/responses.test.js

```diff
@@ -0,0 +1,70 @@
+import { fromJS } from 'immutable';
+
+import {
+  responses,
+  setSportsTimestamp,
+  SET_SPORTS_TIMESTAMP,
+  setLeaguesTimestamp,
+  SET_LEAGUES_TIMESTAMP
+} from './responses';
+
+describe('Given the responses reducer', () => {
+  describe('given setSportsTimestamp action', () => {
+    const res = setSportsTimestamp(1);
+    it('should return an object with type SET_SPORTS_TIMESTAMP', () => {
+      expect(res.type).toBe(SET_SPORTS_TIMESTAMP);
+    });
+
+    it('and should return an object with timestamp payload', () => {
+      expect(res.payload).toBe(1);
+    });
```

Comment:
It can be done with one with one test
`expect(setSportsTimestamp()).toEqual({ type: SET_SPORTS_TIMESTAMP, payload: 1 });`